android/src/main/java/dev/openfeature/sdk/NoOpProvider.kt

```diff
@@ -65,5 +65,7 @@ class NoOpProvider(override val hooks: List<Hook<*>> = listOf()) : FeatureProvid
 
     override fun isProviderReady(): Boolean = true
```

Comment:
should we make this an extension instead which reads the `getProviderStatus` and returns `getProviderStatus() == OpeFeatureEvents.ProviderReady`?

Reply:
Sounds good. I'll fix!